src/ai/backend/manager/models/minilang/ordering.py

```diff
@@ -22,23 +23,31 @@
 
 
 class QueryOrderTransformer(Transformer):
-    def __init__(self, sa_table: sa.Table, column_map: Mapping[str, str] = None) -> None:
+    def __init__(self, sa_table: sa.Table, column_map: Optional[Mapping[str, Any]] = None) -> None:
         super().__init__()
         self._sa_table = sa_table
         self._column_map = column_map
 
     def _get_col(self, col_name: str) -> sa.Column:
         try:
             if self._column_map:
-                col = self._sa_table.c[self._column_map[col_name]]
+                col_value = self._column_map[col_name]
+                if isinstance(col_value, tuple) or isinstance(col_value, list):
+                    if len(col_value) == 1:
+                        col = col_value[0]
+                    else:
+                        func = col_value[1]
+                        col = func(col_value[0])
+                else:
+                    col = self._sa_table.c[self._column_map[col_name]]
```

Comment:
This branch seems not good.
Should I change all `_queryorder_colmap` from `Mapping[str, str]` into `Mapping[str, tuple]`?